### ISSUE TITLE
Seed 0 don't work

### ISSUE THREAD

**Issue by rafbat2591** [toolbox4minecraft/amidst#805]
By typing the number 0 or the related alphanumeric codes, a different seed appears

**Comment by JuniorJedi490** [comment]
I suspect that this is intended behavior as entering the seed '0' in Minecraft generates a random seed.
However, in Minecraft you can still get a world with a seed of 0 by using a text seed, so if text seeds that would produce a seed of 0 are producing a random seed as well, there might be a bug involved.

**Comment by rafbat2591** [comment]
so if I now wanted to see seed 0 as shown here in version 4.5 of amidst, can't I see it in any way?
![screenshot_default_0](https://user-images.githubusercontent.com/67910235/86794473-db82ba00-c06c-11ea-8b47-6e2363c79f96.png)

**Comment by rafbat2591** [comment]
ps: until 1.15.2 if I entered these
-tainsntoh
-f5a5a608
-af4e962cd
it gave me the seed above

**Comment by BigAlanM** [comment]
Verified  using a 1.12.2 profile 0 (zero) produces the 0 (zero)  biome map, bypassing the standard Minecraft action of creating a random world when Seed is either blank or zero.  This is also true for 1.13.2, and 1.14.2.
Only 1.16.2 doesn't perform as expected.

I've know of several text strings that Java's hashcode() would turn into 0 (zero) and bypass the Minecraft call to random.   I  wasn't aware of "tainsntoh", "f5a5a608", "af4e962cd".  
The text strings that I know hash to zero and are short enough to be entered as a seed are:
pollinating sandboxes
amusement & hemophilias
schoolworks = perversive
electrolysissweeteners.net
constitutionalunstableness.net
grinnerslaphappier.org
BLEACHINGFEMININELY.NET
WWW.BUMRACEGOERS.ORG
WWW.RACCOONPRUDENTIALS.NET
bequirtle zorillo
chronogrammic schtoff
contusive cloisterlike
creashaks organzine
drumwood boulderhead
electroanalytic exercisable
favosely nonconstruable

For some reason Amidst  code isn't bypassing the zero equals random code for a 1.16.2 profile. 
Strangely entering "pollinating sandboxes" as a seed to create a 1.16.2 world WILL produce the ZERO seed world.
![1 16 2 zero seed](https://user-images.githubusercontent.com/5558852/86829286-3bc02e80-c062-11ea-973b-4c955a5227dc.png)

**Comment by BigAlanM** [comment]
I did some more testing.  
As I stated, I did create the Zero seed world in MC 1.16.1 and got the correct world.
In  fact, entering /seed while in game will produce a response of Seed: [0].

HOWEVER.  Telling Amidst 4.5 Beta1 to Open Save Game and point it at that Zero Seed world folder will cause Amidst to produce a random world map EVEN THOUGH IT DISPLAYS THE SEED AS 0.  And each time the world file is opened a different world map is displayed.
![Zero world Minutor](https://user-images.githubusercontent.com/5558852/87046956-d7b77a80-c1c7-11ea-81e8-dbdac105ad89.jpg)
![screenshot_default_0 1151](https://user-images.githubusercontent.com/5558852/87046960-d8501100-c1c7-11ea-85a4-4b426841c3ab.png)
